### PR TITLE
Pin Out Reference Image/Colors

### PR DESCRIPTION
**Pull request by Nozemi**
- Pin out reference image, with colors to map the original console cable's wires.
- Added a color column to the pin out table.
- Moved the images into their own directory.
- Updated the image reference in the firmware.rst file so it references the new image location.

### CODE REVIEW

**Comment by nomis** [comment]
Could you use SVG instead of JPEG for `images/console-port-diagram.jpg`?

File: interface.rst [hunk 1]
```diff
@@ -9,39 +9,46 @@ cable (but the screws to hold the connectors together will be mismatched).
...
+|  14 |  Orange       | Unused                                    | Unused                  |
++-----+---------------+-------------------------------------------+-------------------------+
+|  15 |  Red/White    | 0V if present                             | Console presence        |
++-----+---------------+-------------------------------------------+-------------------------+
+
+.. image:: images/console-port-diagram.jpg
+	:height: 156px
+	:alt: Console port pinout reference image.
+
+*Image reference of the port, showing which wire color is what on the original console cable.*
```
Comment (nomis) [inline interface.rst:50]: You need to indent this with 3 spaces so that it's a caption and not a separate paragraph.

Comment (nomis) [reply interface.rst:50]: (I don't think it needs italics)

File: interface.rst [hunk 2]
```diff
@@ -9,39 +9,46 @@ cable (but the screws to hold the connectors together will be mismatched).
...
+**NOTE! This is from the sub/amp side, not the numbers on the cable!** *(yes the connector on the cable is labeled with numbers)*
```
Comment (nomis) [inline interface.rst:51]: I'd just state that the connector is labelled with numbers that are different from those of the connector. It should be implied that these are the DB-15 pin numbers...

**Comment by nomis** [comment]
The pins in the diagram are from the perspective of the amplifier connector but the colours are for the console cable which has the pins in the mirrored order...

**Review by nomis — CHANGES_REQUESTED** [review]

**Comment by nomis** [comment]
The colours in here don't match those on https://www.reddit.com/r/hardwarehacking/comments/hnpprk/hacking_the_logitech_z906_speaker_system/.

There's no reason why Logitech would have to use the same colour wires on every batch so it's possible they're not always the same. There are two variants of the console - with and without an exposed programming port - so these could have different wire colours too.

**Comment by Nozemi** [comment]
> 

I noticed that the wiring colors aren't matching, not sure what's up with that. Haven't really had time to look into it either, so I haven't done anything more with this pull request.

I have both of the consoles at hand, so I suppose I could check to see if there is a difference between them.

However, the wiring diagram on the Reddit link seems to work, at least for the cable I cut off the connectors of.

Closing this PR, will open a new one if I can find time to produce something definitively correct, and useful.

**Comment by nomis** [comment]
> However, the wiring diagram on the Reddit link seems to work, at least for the cable I cut off the connectors of.

I don't recommend cutting the cables because they use a standard connector... I suggest you get a [VGA extension cable](https://en.wikipedia.org/wiki/VGA_connector) next time and cut that in half. The colours of the wires are only useful if you're cutting cables otherwise the pin numbering is sufficient.

**Pull request closed without merging**